apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -643,14 +648,14 @@ export class CaseNotificationService extends BaseNotificationService {
           RequestSharedWithDefender.COURT_DATE,
     )
 
-    return this.sendEmail(
-      linkSubject,
-      linkHtml,
-      theCase.defenderName,
-      theCase.defenderEmail,
-      undefined,
-      !theCase.defenderNationalId,
-    )
+    return this.sendEmail({
+      subject: linkSubject,
+      html: linkHtml,
+      recipientName: theCase.defenderName,
+      recipientEmail: theCase.defenderEmail,
+      attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.